spec/core_functions/math/atan2/arguments.hrx

```diff
@@ -0,0 +1,138 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#3044
```

Comment:
Rather than giving each subdirectory of `math/atan2` an identical `options.yml` file, just put one directly in `math/atan2`. It'll apply to everything beneath it.